package.json

```diff
@@ -44,8 +45,13 @@
   "resolutions": {
     "elliptic@6.5.4": "^6.5.7",
     "fast-xml-parser@^4.3.4": "^4.4.1",
+    "react-native": "^0.75.4",
     "ws@7.4.6": "^7.5.10"
   },
+  "dependencies": {
+    "react": "^18.3.1",
+    "react-native": "^0.75.4"
+  },
```

Comment:
Should these be in devDependencies?

Reply:
should be in devDependencies and peerDependencies